src/NzbDrone.Core/NetImport/TMDb/TMDbSettings.cs

```diff
@@ -69,6 +69,10 @@ public TMDbSettings()
         [FieldDefinition(0, Label = "TMDb API URL", HelpText = "Link to to TMDb API URL, do not change unless you know what you are doing.")]
         public string Link { get; set; }
 
+        [FieldDefinition(0, Label = "Account ID", HelpText = "Your account ID for Radarr API.")]
```

Comment:
Field Definitions cannot have the same number! I would move this down to be the last item. At least further down than the list type.

Reply:
thought I fixed that one - will change it.